dashboard/package.json

```diff
@@ -26,7 +26,7 @@
     "watch-css": "yarn run build-css && yarn run compile-lang && sass src/ --watch --no-source-map"
   },
   "dependencies": {
-    "@babel/parser": "^7.20.15",
+    "@babel/parser": "^7.18.4",
```

Comment:
Please note this change won't take effect unless the manual resolution under `resolutions` is also upgraded 😅